mission-racing/src/main/java/com/racing/model/Cars.java

```diff
@@ -1,50 +1,67 @@
 package com.racing.model;
 
+import com.racing.exception.CarNotFoundException;
+import com.racing.exception.NameLengthException;
+
 import java.util.*;
 import java.util.stream.Collectors;
 
 public class Cars {
 
-    private List<Car> carlist;
+    private static final int NAME_MAX_LENGTH = 5;
+    private List<Car> cars = new ArrayList<>();
```

Comment:
```suggestion
    private static final int NAME_MAX_LENGTH = 5;
    
    private List<Car> cars = new ArrayList<>();
```

cars를 작성하신 코드처럼 바로 초기화해주는 것과 생성자를 통해 초기화해주는 것의 차이가 무엇이고, 어떤 장단점이 있을까요

Reply:
학습하면서, 초기화 값이 객체가 생성될때 동적으로 결정되어야 하는 경우에는 생성자를 통해 초기화 하는것이 유용하다는 점을 알게 되었습니다!
항상 동일한 초기값을 가져야 하는 경우에는 변수를 선언하면서 초기화 하는것이 적합한 것 같습니다.